components/zcash_address/Cargo.toml

```diff
@@ -24,3 +23,5 @@ proptest = "1"
 
 [lib]
 bench = false
+
+[features]
```

Comment:
Please remove this empty section from your commit (I presume they were added automatically by the tool):
```suggestion
```